llvm/test/CodeGen/DirectX/rootsignature-validation.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: opt -S -passes='dxil-post-optimization-validation' -mtriple=dxil-pc-shadermodel6.6-compute %s 2>&1 
```

Comment:
Why pipe stderr to stdout here? The `2>&1` is more likely to make things more confusing than help.